packages/components/src/components/icon/flag/ic-flag-ar.svg

```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="24" height="17" fill="none"><mask id="a" width="24" height="17" x="0" y="0" maskUnits="userSpaceOnUse" style="mask-type:alpha"><path fill="#fff" d="M22 .406H2a2 2 0 0 0-2 2v12a2 2 0 0 0 2 2h20a2 2 0 0 0 2-2v-12a2 2 0 0 0-2-2Z"/></mask><g mask="url(#a)"><path fill="#FF323E" d="M22 .406H2a2 2 0 0 0-2 2v12a2 2 0 0 0 2 2h20a2 2 0 0 0 2-2v-12a2 2 0 0 0-2-2Z"/><path fill="#262626" d="M6 11.406h18v5H6v-5Z"/><path fill="#fff" d="M6 5.406h18v6H6v-6Z"/><path fill="#12833B" d="M6 .406h18v5H6v-5Z"/><path fill="#000" fill-opacity=".08" fill-rule="evenodd" d="M22 1.406H2a1 1 0 0 0-1 1v12a1 1 0 0 0 1 1h20a1 1 0 0 0 1-1v-12a1 1 0 0 0-1-1Zm-20-1a2 2 0 0 0-2 2v12a2 2 0 0 0 2 2h20a2 2 0 0 0 2-2v-12a2 2 0 0 0-2-2H2Z" clip-rule="evenodd"/></g></svg>
```

Comment:
@suisin-deriv please raise a concern to the PD team to have a generic flag for Arabic language than a specific country's flag.